### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by SuhaibMujahid**
```
go get gopkg.in/src-d/enry.v1/...: unrecognized import path "gopkg.in/src-d/enry.v1/..." (parse https://gopkg.in/src-d/enry.v1/...?go-get=1: no go-import meta tags ())
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Installation
 The recommended way to install enry is
 
 ```
-go get gopkg.in/src-d/enry.v1/...
+go get gopkg.in/src-d/enry.v1
```
Comment (bzz) [inline README.md]:
```suggestion
go get gopkg.in/src-d/enry.v1/cmd/enry
```

**Comment by bzz** [comment]
Thank you for the patch @SuhaibMujahid!
What do you think about changing it as in https://github.com/src-d/enry/pull/211#discussion_r271303916  ?

**Comment by SuhaibMujahid** [comment]
Thank you @bzz  for consideration.

I tried it but it didn't work:
```
go get gopkg.in/src-d/enry.v1/cmd/enry: invalid module path "gopkg.in/src-d/enry.v1/cmd/enry"
```

I'm trying to use `enry` as a dependency in my application and the modification that I proposed worked for me.

Have a good day :)

**Comment by creachadair** [comment]
> Thank you @bzz for consideration.
> 
> I tried it but it didn't work:
> 
> ```
> go get gopkg.in/src-d/enry.v1/cmd/enry: invalid module path "gopkg.in/src-d/enry.v1/cmd/enry"
> ```
> I'm trying to use `enry` as a dependency in my application and the modification that I proposed worked for me.

Hmm, that error surprises me. What version of Go are you currently building with (`go version`)?

**Comment by SuhaibMujahid** [comment]
I'm using `go1.11.5 darwin/amd64`with versioned modules.

**Comment by creachadair** [comment]
Thanks. I am unfortunately having some trouble reproducing this issue right now. Here's what I tried—if you run this locally does it work?

```sh
# Acquire Go 1.11.5
go get golang.org/dl/go1.11.5
go1.11.5 download
# …
go1.11.5 clean -modcache

# First, let's try it without modules.
rm $GOPATH/bin/enry
env -u GO111MODULE go1.11.5 get gopkg.in/src-d/enry.v1/cmd/enry
# works OK
enry -version
# undefined (OK)

# Now let's try it with modules enabled.
rm $GOPATH/bin/enry
mkdir testing ; cd testing ; go mod init testing
enry -version
# undefined (OK)
```

**Comment by SuhaibMujahid** [comment]
After running `clean -modcache`  it worked.
```bash
go get gopkg.in/src-d/enry.v1/cmd/enry
# go: downloading gopkg.in/src-d/enry.v1 v1.7.2
# go: downloading gopkg.in/toqueteos/substring.v1 v1.0.2
# go: downloading github.com/toqueteos/trie v1.0.0
enry -version
# undefined
```
Thank you.

**Comment by creachadair** [comment]
Thanks. I suspect you might have had a broken module hash somewhere from an earlier 1.11 build (cf. https://github.com/golang/go/issues/27093).

**Pull request merged**